ynput/OpenPype
# Feature: Resolve: Open last workfile on launch through .scriptlib

## Changelog Description

Added implementation to Resolve integration to open last workfile on launch.

## Additional info

This proposes an alternative implementation to what https://github.com/ynput/OpenPype/pull/5006 tries to solve.

Also added the option to directly launch the OpenPype menu on launch. Still have to add a toggle in settings to enable/disable that based on studio settings.

## Testing notes:

1. Launch resolve. It should open the last workfile directly if it exists (and open last workfile setting must be enabled, which is the default setting)

Or for a complete reproducable:

1. Select a task in a project
2. Launch Resolve on the task
3. Save the workfile using workfiles tool
4. Close resolve
5. Launch it again on the same task
6. The workfile should be opened on resolve launch
